Cesium.CodeGen.Tests/verified/CodeGenArrayTests.UnSignedByteArrayTest2.verified.txt

```diff
@@ -1,37 +1,49 @@
-﻿System.Int32 <Module>::main()
-  Locals:
-    System.Byte* V_0
-    System.Byte V_1
-    System.Int32 V_2
-  IL_0000: ldc.i4.1
-  IL_0001: conv.u
-  IL_0002: localloc
-  IL_0004: stloc.0
-  IL_0005: ldsflda <ConstantPool>/<ConstantPoolItemType1> <ConstantPool>::ConstDataBuffer0
-  IL_000a: ldloc V_0
-  IL_000e: ldc.i4.1
-  IL_000f: conv.u
-  IL_0010: call System.Void Cesium.Runtime.RuntimeHelpers::InitializeCompound(System.Void*,System.Void*,System.UInt32)
-  IL_0015: ldloc.0
-  IL_0016: ldc.i4.0
-  IL_0017: conv.i
-  IL_0018: ldc.i4 1
-  IL_001d: mul
-  IL_001e: add
-  IL_001f: ldind.u1
-  IL_0020: stloc.1
-  IL_0021: ldloc.1
-  IL_0022: conv.i4
-  IL_0023: stloc.2
-  IL_0024: ldloc.2
-  IL_0025: ret
+﻿Module: Primary
+  Type: <Module>
+  Methods:
+    System.Int32 <Module>::main()
+      Locals:
+        System.Byte* V_0
+        System.Byte V_1
+        System.Int32 V_2
+      IL_0000: ldc.i4.1
+      IL_0001: conv.u
+      IL_0002: localloc
+      IL_0004: stloc.0
+      IL_0005: ldsflda <ConstantPool>/<ConstantPoolItemType1> <ConstantPool>::ConstDataBuffer0
+      IL_000a: ldloc V_0
+      IL_000e: ldc.i4.1
+      IL_000f: conv.u
+      IL_0010: call System.Void Cesium.Runtime.RuntimeHelpers::InitializeCompound(System.Void*,System.Void*,System.UInt32)
+      IL_0015: ldloc.0
+      IL_0016: ldc.i4.0
+      IL_0017: conv.i
+      IL_0018: ldc.i4 1
+      IL_001d: mul
+      IL_001e: add
+      IL_001f: ldind.u1
+      IL_0020: stloc.1
+      IL_0021: ldloc.1
+      IL_0022: conv.i4
+      IL_0023: stloc.2
+      IL_0024: ldloc.2
+      IL_0025: ret
 
-System.Int32 <Module>::<SyntheticEntrypoint>()
-  Locals:
-    System.Int32 V_0
-  IL_0000: call System.Int32 <Module>::main()
-  IL_0005: stloc.s V_0
-  IL_0007: ldloc.s V_0
-  IL_0009: call System.Void Cesium.Runtime.RuntimeHelpers::Exit(System.Int32)
-  IL_000e: ldloc.s V_0
-  IL_0010: ret
+    System.Int32 <Module>::<SyntheticEntrypoint>()
+      Locals:
+        System.Int32 V_0
+      IL_0000: call System.Int32 <Module>::main()
+      IL_0005: stloc.s V_0
+      IL_0007: ldloc.s V_0
+      IL_0009: call System.Void Cesium.Runtime.RuntimeHelpers::Exit(System.Int32)
+      IL_000e: ldloc.s V_0
+      IL_0010: ret
+
+  Type: <ConstantPool>
+  Nested types:
+    Type: <ConstantPool>/<ConstantPoolItemType1>
+    Pack: 1
+    Size: 1
+  Fields:
+    <ConstantPool>/<ConstantPoolItemType1> <ConstantPool>::ConstDataBuffer0
+      Init with (UTF-8 x 1 bytes): "�"
```

Comment:
Should we write this as escape sequences to improve validation?

Reply:
Yeah, would be great. I improved it a bit but skipped these so far.